tinyagentos/routes/desktop_browser/copilot_agent_ws.py

```
@@ -156,6 +217,28 @@ async def copilot_agent_ws(websocket: WebSocket, ticket: str):
+                    try:
+                        asyncio.create_task(_maybe_send_chat_push(
+                            user_id=consumed.user_id,
+                            agent_id=consumed.agent_id,
+                            agent_name=consumed.agent_id,
+                            msg_text=msg_text,
+                            window_id="",
+                            tab_id=target_tab,
+                            store=store,
+                            hub=hub,
+                            vapid=vapid,
+                        ))
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)